modules/Bio/EnsEMBL/Analysis/Hive/Config/TranscriptSelection.pm

```diff
@@ -0,0 +1,910 @@
+    'skip_rnaseq'               => '0',                                                         # Will skip rnaseq analyses if 1
+    'skip_cleaning'             => '0',                                                         # Will skip the cleaning phase, will keep more genes/transcripts but some lower quality models may be kept
+
+########################
+# Pipe and ref db info
+########################
+
+    'projection_source_db_name'         => '',                                                  # This is generally a pre-existing db, like the current human/mouse core for example
+    'projection_source_db_server'       => 'mysql-ens-mirror-1',
+    'projection_source_db_port'         => '4240',
+    'projection_source_production_name' => '',
+
+    'cdna_db_server' => $self->o('databases_server'),
+    'cdna_db_port'   => $self->o('databases_port'),
+
+    'genblast_db_server' => $self->o('databases_server'),
+    'genblast_db_port'   => $self->o('databases_port'),
+
+    'projection_db_server' => $self->o('databases_server'),
+    'projection_db_port'   => $self->o('databases_port'),
+
+    'long_read_final_db_server' => $self->o('databases_server'),
+    'long_read_final_db_port'   => $self->o('databases_port'),
+
+    'rnaseq_for_layer_db_server' => $self->o('databases_server'),
+    'rnaseq_for_layer_db_port'   => $self->o('databases_port'),
+
+    # Layering is one of the most intesnive steps, so separating it off the main output server helps
+    # Have also set module to use flatfile seq retrieval, so even if it's on the same server as the
+    # core, the core should not be accessed
+    'layering_db_server' => $self->o('dna_db_server'),
+    'layering_db_port'   => $self->o('dna_db_port'),
+
+    'utr_db_server' => $self->o('databases_server'),
+    'utr_db_port'   => $self->o('databases_port'),
+
+    'genebuilder_db_server' => $self->o('databases_server'),
+    'genebuilder_db_port'   => $self->o('databases_port'),
+
+    'pseudogene_db_server' => $self->o('databases_server'),
+    'pseudogene_db_port'   => $self->o('databases_port'),
+
+    'ncrna_db_server' => $self->o('databases_server'),
+    'ncrna_db_port'   => $self->o('databases_port'),
+    ncrna_db_name     => $self->o('dbowner') . '_' . $self->o('production_name') . '_ncrna_' . $self->o('release_number'),
+
+    'final_geneset_db_server' => $self->o('databases_server'),
+    'final_geneset_db_port'   => $self->o('databases_port'),
+
+    # This is used for the ensembl_production and the ncbi_taxonomy databases
+    'ensembl_release'      => $ENV{ENSEMBL_RELEASE},     # this is the current release version on staging to be able to get the correct database
+    'production_db_server' => 'mysql-ens-meta-prod-1',
+    'production_db_port'   => '4483',
+
+    databases_to_delete => [ 'cdna_db', 'genblast_db', 'selected_projection_db', 'layering_db', 'utr_db', 'genebuilder_db', 'pseudogene_db', 'ncrna_db', 'final_geneset_db' ],    #, 'projection_realign_db'
```

Comment:
Only databases created in the pipeline should be in this array so only layering_db, utr_db, genebuild_db, peusdogene_db and final_geneset_db should be in the array. Otherwise we would delete databases created by other pipelines which could be long to produce if using `-drop_databases 1` on the command line during initialisation